### ISSUE TITLE
Should we refactor the venue mapping to be a config file?

### ISSUE THREAD

**Issue by jdonkervliet** [atlarge-research/venue_mappings#32]
I was wondering why the mapping is implemented in Python. It seems to be more data than code. Should we refactor the mapping to be a Yaml file of some kind? The befenits would be:

1. Simplifies importing the mapping. Beautifybib now assumes this Python module is present, managed by pip. This restricts how you can access the mapping. Instead, it can be located in (e.g.,) `~/.beautifybib.conf`
2. Simplifies editing the mapping. If the file is managed by pip, users should really not touch it. If it's a config file, editing is encouraged.
3. If it is a config file, we can implement a feature in Beautifybib to automatically download the file, simplifying further the user experience.

**Comment by lfdversluis** [comment]
There are two projects that use the venue mapper, one is beautifybib (closed source) and the other is AIP (open source). I would keep it in a separate repo (i.e., here) as you would only have to maintain it in one place then.

**Comment by lfdversluis** [comment]
I agree that having to install it via pip is less than ideal, if you have better ideas, shoot! :D
